src/main/resources/application.yaml

```diff
@@ -1,12 +1,6 @@
 server:
   port: 4000
 
-logging:
```

Comment:
This intentional?

Reply:
yes, it was put there to debug idam issues that are now resolved